.github/workflows/lint.yaml

```diff
@@ -14,3 +14,14 @@ jobs:
           python-version: '3.11'
       - run: pip install ruff
       - run: ruff check beangulp/ examples/
+  mypy:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.9'
```

Comment:
Why Python 3.9?

Reply:
that's the oldest supported version of Beancount - which version do you want?

Reply:
If it helps to narrow down the choices: 3.9 is the oldest version with security-only support, but 3.12 is the oldest with [full support](https://peps.python.org/pep-0693/).